core/src/main/java/org/stellar/anchor/util/MemoHelper.java

```
@@ -76,4 +73,43 @@ public static Memo makeMemo(String memo, MemoType memoType) throws SepException
   public static String convertBase64ToHex(String memo) {
     return Hex.encodeHexString(Base64.getDecoder().decode(memo.getBytes()));
   }
+
+  public static String memoAsString(Memo memo) {
+    switch (getMemoType(memo)) {
+      case MEMO_NONE:
+        return null;
+
+      case MEMO_ID:
+        return String.valueOf(((MemoId) memo).getId());
+
+      case MEMO_TEXT:
+        return ((MemoText) memo).getText();
+
+      case MEMO_HASH:
+        return ((MemoHash) memo).getHexValue();
+
+      case MEMO_RETURN:
+        return ((MemoReturnHash) memo).getHexValue();
+    }
+
+    return null;
```

Should we throw exception instead of null?